src/Stratis.Bitcoin.Features.RPC.Tests/Controller/RPCControllerTest.cs

```diff
@@ -46,14 +46,13 @@ public RPCControllerTest()
             this.fullNode.Setup(f => f.Network)
                 .Returns(this.testNetwork);
             this.rpcHost = new Mock<IWebHost>();
-            var nodeSettings = new NodeSettings(this.testNetwork, args: new string[] { "-server=1" });
+            var nodeSettings = new NodeSettings(this.testNetwork, args: new string[] { "-server=1", "-rpcuser=test", "-rpcpassword=test", "-rpcport=0" });
```

Comment:
Are you adding the extra params here cause now if you specify server=1 then user and password is required?

Reply:
Yes.